mussel

```diff
@@ -559,6 +561,33 @@ while [ $# -gt 0 ]; do
   g | -g | --disable-isl)
     ISL_SUPPORT=no
     ;;
+  T | -T | --custom-vendor)
+    # Have the user defined the default tuple first?
+    if [ -z $XTARGET ]; then
+      printf '%b!!%b '\''%s'\'' requires that the target architecture is already specified.\n' \
+        "$REDC" "$NORMALC" $1
+      printf 'Run '\''$0 -h'\'' for help.\n' $0
+      exit 1
+    fi
+    # Set the vendor as the next argument to this flag.
+    shift
+    CUSTOM_VENDOR="$1"
+    if (echo "$CUSTOM_VENDOR" | grep ' ') 2>&1 >/dev/null; then
```

Comment:
@firasuke I think I will rewrite this part to not use grep.

Reply:
Personally I don't think that will be necessary as `grep` is expected to be on most systems where `mussel` will be run.